posthog/temporal/runner.py

```diff
@@ -0,0 +1,15 @@
+from datetime import timedelta
+
+from temporalio.client import Client
+
+from posthog.temporal.workflows import NoOpWorkflow
+
+
+async def execute_noop_workflow(host, port):
+    client = await Client.connect(f"{host}:{port}")
+    result = await client.execute_workflow(
+        NoOpWorkflow.run,
+        start_to_close_timeout=timedelta(seconds=60),
+        schedule_to_close_timeout=timedelta(minutes=5),
+    )
```

Comment:
I believe this call is missing the `time` argument for `NoOpWorkflow.run`.

Reply:
wooops!